README.adoc

```diff
@@ -139,3 +139,47 @@ https://api.studio.thegraph.com/query/24143/threshold/0.0.3
   }
 }
 ```
+
+== Test the subgraph locally
+
+=== Run local chain eg. Ganache
+
+Note that host set to `0.0.0.0` is necessary for Ganache to be accessible from
+within Docker and from other machines. By default, Ganache only binds to
+127.0.0.1, which can only be accessed from the host machine that Ganache runs
+on. If you use a ganache-cli run the node with `ganache-cli -h 0.0.0.0`. If you
+use a Ganche in GUI version go to `settings -> server ` and set `HOSTNAME` to
+`0.0.0.0 - All Interfaces`.
+
+=== Deploy the Threshold Network contracts to local chain
```

Comment:
Following these steps are not enough to have running the local Graph Node. In addition to these, we should write here that, in order to deploy the local Graph Node, it's necessary:
- To have `keep-core` contracts deployed (and how to do this).
OR
- To modify deployment scripts of `solidity-contract` repo to deploy the stubs.

IMHO these steps are necessary to be written here, especially for those of us who are not familiar with keep-core contracts deployment.

Reply:
22da4ff